src/lib/components/Sidebar.svelte

```diff
@@ -74,6 +76,21 @@ The sidebar/drawer.
             <!-- svelte-ignore a11y-click-events-have-key-events -->
             <li><a on:click={closeDrawer}><TextAppearanceIcon />Text Appearance</a></li>
             <div class="dy-divider m-1" />
+            {#if menuItems}
+                {#each menuItems as item}
+                    <li>
+                        <a href={item.link['default']}>
+                            <img
```

Comment:
Did you try to use the `<picture>` element that can handle multiple resolutions (especially since the extra icons are 2x and 3x the size).